### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_AzureNotebooksWelcome_Inspect:The BoundingRectangle must represent a rectangle with sufficient height, width, and area. 

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3990]
**Environment Details:**
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 68.0.3440.106
Operating System:Windows10

**Repro Steps:**
1.Login with the appropriate credentials to azure notebook account.
2.Navigate to run button & hit enter --> Navigate to Azure Notebooks welcome & hit enter.
3.Verify Name, role, value & State of controls present on page.

**Actual:**
The BoundingRectangle must represent a rectangle with sufficient height, width, and area. 
The separator does not identify separation of content on page.

**Expected:**
BoundingRectangle value should be correct.

**User Impact:**
User is unable to identify the page content.

**Attachment:**
![a11y_azurenotebooks_jupyter_azurenotebookswelcome_inspect_value](https://user-images.githubusercontent.com/39204909/45808866-78cbfd00-bce4-11e8-839a-6dd2dad0eb56.png)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
